modules/monitoring-managing-user-permissions-using-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="managing-user-permissions-using-cli_{context}"]
+= Managing user permissions using CLI
+
+This procedure shows how to manage user permissions for monitoring their own services using the CLI.
+
+.Prerequisites
+
+* You need to have a user created.
+* Log in using the `oc` command.
+
+.Procedure
+
+This example shows how to assign permissions to user `johnsmith` for setting up metrics collection and creating alerting rules in the `ns1` namespace:
+
+* Run this command to assign <role> to <user> in <namespace>:
++
+----
+$ oc policy add-role-to-user <role> <user> -n <namespace>
+----
++
+Substitute <role> for `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.
++
+[IMPORTANT]
+====
+Whichever role you choose, you must bind it against a specific namespace as a cluster administrator.
+====
++
+--
+** `monitoring-rules-view` allows reading PrometheusRule custom resources within the namespace.
+** `monitoring-rules-edit` allows creating, modifying, and deleting PrometheusRule custom resources matching the permitted namespace.
+** `monitoring-edit` gives the same permissions as `monitoring-rules-edit`. Additionally, it allows creating new scraping targets for services or pods. It also allows creating, modifying, and deleting ServiceMonitors and PodMonitors.
+--
++
+In this example, substitute <role> for `monitoring-edit`, <user> for `johnsmith`, and <namespace> for `ns1`.
```

Comment:
I'm confused by this as well. By "In this example", I think you mean in the code block example but I'm not certain. Also, previously, it says to `Substitute <role> for` one of three options, but here it is saying to substitute it with `monitoring-edit` - so which is suggested?

I'm wondering if the three roles should be another annotation?

Also, instead of `substitute <role> _for_`, I would say, `substitute <role> _with_`.

Reply:
I've changed this, it should be more clear now. Again, how would you use an annotation here?

Reply:
Looks much better now. And saying "As an example" is clearer to me than "In this example". The latter made me think it was something the user needed to do. Ignore my comment about the annotation.